pkg/sqlite/directory.go

```diff
@@ -128,10 +182,56 @@ func (d *DirectoryLoader) LoadBundleWalkFunc(path string, f os.FileInfo, err err
 	return utilerrors.NewAggregate(errs)
 }
 
-// LoadBundle takes the directory that a CSV is in and assumes the rest of the objects in that directory
+// LoadPackagesWalkFunc attempts to unmarshal the file at the given path into a PackageManifest resource.
+// If unmarshaling is successful, the PackageManifest is added to the loader's store.
+func (d *DirectoryLoader) LoadPackagesWalkFunc(path string, f os.FileInfo, err error) error {
+	if f == nil {
+		return fmt.Errorf("invalid file: %v", f)
+	}
+
+	log := logrus.WithFields(logrus.Fields{"dir": d.directory, "file": f.Name(), "load": "package"})
+	if f.IsDir() {
+		if strings.HasPrefix(f.Name(), ".") {
+			log.Info("skipping hidden directory")
+			return filepath.SkipDir
+		}
+		log.Info("directory")
```

Comment:
Should this be a debug statement? It doesn't seem to mean anything from a user perspective.

Reply:
The log will have all of the fields added to it, so the output shows us visiting each node.